paper.md

```diff
@@ -86,14 +86,63 @@ Bullet point version (can be removed after discussion):
 * Course contains stand-alone lectures that outline broadly useful statistical methods.
 * Modular: lectures can be mixed and matched
 * This course is completely interactive: all lectures are designed to be delivered in a participatory live-coding format so that students learn experientally in real-time.
-* This course contains assignments that sharpen students' understanding and allow them to critically apply skills to new problems.  
+* This course contains assignments that sharpen students' understanding and allow them to critically apply skills to new problems.
 * Students learn creativity and critical thinking through an open-ended, self-directed project that uses real ecological data.
 * Students learn skills for reproducible research throughout the course.
 * Course website - nice looking and easy to use
 * We have taught this course twice and have refined the materials based on previous teaching experience
 
 # Main Body
 
+## Instructional Design
+
+Drawing from the authors' previous experiences teaching introductory programming
+workshops, we designed each of our lectures to have the following components:
+
+    1. *Lesson Outline*: A clearly defined outline of the lesson objectives including expected 
+    length of time spent on each objective.
+        - This makes clear to students not only what they can expect to learn from the lecture,
+        but also provides a structured template for instructors to prioritize content and gauge
+        how long they should be spending on each objective. This sets the expectations for both
+        students and instructors.
+    3. *Participatory Live-Coding*: Coding snippets that form the teaching content of each lesson 
+    objective, taught in-class using live-coding.[^1]
+       - These submodules go over all concepts necessary to understand the concepts, complete the
```

Comment:
"concepts" used twice. Maybe change second one to "course materials" or something similar?